### ISSUE TITLE
Request to Determine Institutional & Funding Standards

### ISSUE THREAD

**Issue by LPaulson1985** [ProgressiveCoders/functions#271]
#### Description

...

#### Problem

...

#### Benefit

...

#### Plan

- [ ]
...[](https://docs.google.com/document/d/1mi8qbuB-yROAcRvhEWdlvXbTB0jGOqygzPL_IO1McI0/edit?usp=sharing)

#### Decision Making

...

#### Reference link(s)

...

**Comment by jpb5013** [comment]
Thanks for putting this together, @LPaulson1985! 

I overall agree with the approach to avoid funding from any non mission-aligned organizations and avoiding funding from publicly traded companies. 

The biggest thing I've been thinking about is the revolving door between for profit institution and non-profit foundation and where we draw the line. For example, the Gates Foundation received a significant amount of its funding from Microsoft via Bill Gates, so would we be open to receiving a grant from them?



As @stephenscapelliti mentioned last night, we started down this path in the past, so there may be some previous thinking worth leveraging. There was some discussion around this in Issue #81 and #227 includes a couple useful links (like this [Funding Use Cases](https://docs.google.com/spreadsheets/d/16YkBS2sxCBCPfCxSzibFaEk5xvm_oNoB70FXIl_4XVs/edit#gid=0).

I think my overall perspective is to avoid overengineering things out of the gate and instead to make the guidelines flexible and adjustable as we learn. We may want to consider some overarching requirements though (e.g., refusal to give up community ownership) and I always think it's a good idea to include some opportunity for community input on potential donors (even if it's different than the standard change process).

**Comment by LPaulson1985** [comment]
> 

I agree Joe. This draft was just meant to be a general, community-editable guideline for me, Justin, and anyone else fundraising/grant-seeking on behalf of ProgCode as well as staff working in the week. (Or, bob-willing, someone pops up and wants to hand ProgCode an air baloon full of money or something!!) The reference issues above [81](https://github.com/ProgressiveCoders/functions/issues/81) and [227](https://github.com/ProgressiveCoders/functions/issues/227) are EXTREMELY pertinent to this end. And I am wishing Rapi was here, so I could ask him more about where that landed and what he found in his experience grant-seeking/fundraising. He had some pretty amazing ideas in there regarding, educational membership services, etc. Thank you for linking these in, @jpb5013. The Funding Use Case as well! Very helpful. MAy be worth answering some of the undefined and revisiting to see where the community stands on these funding sources at this point.
